_data-prepper/pipelines/configuration/processors/mutate-event.md

```diff
@@ -3,311 +3,18 @@ layout: default
 title: Mutate event
 parent: Processors
 grand_parent: Pipelines
-nav_order: 45
+nav_order: 65
 ---
 
 # Mutate event processors
 
 Mutate event processors allow you to modify events in Data Prepper. The following processors are available:
 
-* [AddEntries](#addentries) allows you to add entries to an event.
-* [CopyValues](#copyvalues) allows you to copy values within an event.
-* [DeleteEntry](#deleteentry) allows you to delete entries from an event.
-* [RenameKey](#renamekey) allows you to rename keys in an event.
-* [ConvertEntry](#convertentry) allows you to convert value types in an event.
+* [add_entries]({{site.url}}{{site.baseurl}}/data-prepper/pipelines/configuration/processors/add-entries/) allows you to add entries to an event.
+* [copy_values]({{site.url}}{{site.baseurl}}/data-prepper/pipelines/configuration/processors/copy-values/) allows you to copy values within an event.
+* [delete_entries]({{site.url}}{{site.baseurl}}/data-prepper/pipelines/configuration/processors/delete-entries/) allows you to delete entries from an event.
+* [rename_keys]({{site.url}}{{site.baseurl}}/data-prepper/pipelines/configuration/processors/rename-keys/) allows you to rename keys in an event.
+* [convert_entry_type]({{site.url}}{{site.baseurl}}/data-prepper/pipelines/configuration/processors/convert_entry_type/) allows you to convert value types in an event.
```

Comment:
Let's add list-to-map processor to this list as well.

Reply:
Done.